pkg/awsv1alpha1/accounts.go

```diff
@@ -0,0 +1,62 @@
+package awsv1alpha1
+
+import (
+	"context"
+	"fmt"
+	awsv1alpha1 "github.com/openshift/aws-account-operator/pkg/apis/aws/v1alpha1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	awsCredsSecretIDKey     = "aws_access_key_id"
+	awsCredsSecretAccessKey = "aws_secret_access_key"
+	namespace               = "aws-account-shredder"             // change the namespace according to your environment. this is the namespace, from where secret has to retreived from
+	secretName              = "aws-account-shredder-credentials" // the name of the secret to be read
+)
+
+func GetAcoountCRsToReset(ctx context.Context, cli client.Client) ([]string, error) {
+
+	var accounts awsv1alpha1.AccountList
+	err := cli.List(ctx, &accounts, &client.ListOptions{
+		Namespace: "aws-account-operator",
+	})
+	if err != nil {
+		fmt.Println(err)
+		return nil, err
+	}
+
+	var accountID []string
+	for _, account := range accounts.Items {
+		if account.Spec.ClaimLink == "" && account.Status.State == "Failed" {
+			accountID = append(accountID, account.Spec.AwsAccountID)
+		}
+
+	}
+	return accountID, err
+}
+
+func GetAWSAccountCredentials(ctx context.Context, cli client.Client) ([]byte, []byte, error) {
```

Comment:
Why is this function in the `awsv1alpha` package?

Reply:
moved to a different package 'k8sWrapper'